src/widgets/slider.cpp

```diff
@@ -23,12 +23,16 @@ static const int SLIDER_WIDTH = 3;
  * @param r     Rectangle to draw the widget in
  * @param value Value to put the slider at
  */
-void DrawSliderWidget(Rect r, int min_value, int max_value, int value)
+void DrawSliderWidget(Rect r, int min_value, int max_value, int value, std::map<int, StringID> labels)
```

Comment:
`const std::map<int, StringID> &labels` to prevent copying the map for each draw?